src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests/BundleEnvironmentGetCommandLineArgs.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using BundleTests.Helpers;
+using Microsoft.DotNet.Cli.Build.Framework;
+using Microsoft.DotNet.CoreSetup.Test;
+using Xunit;
+
+namespace AppHost.Bundle.Tests
+{
+    public class BundleEnvironmentGetCommandLineArgs : IClassFixture<BundleEnvironmentGetCommandLineArgs.SharedTestState>
+    {
+        private SharedTestState sharedTestState;
+
+        public BundleEnvironmentGetCommandLineArgs(SharedTestState fixture)
+        {
+            sharedTestState = fixture;
+        }
+
+        [Fact]
+        public void GetEnvironmentArgs_0_Returns_Bundled_Executable_Path()
```

Comment:
Its probably a good idea to add a test for non-single-file case here, and check that we get `AppPath` -- to reinforce our understanding, given that we have the test infrastructure available.